setup.py

```diff
@@ -3,10 +3,7 @@
 import re
 import sys
 import warnings
-try:
-    from setuptools import setup
-except:
-    from distutils.core import setup
+from setuptools import setup, find_packages
```

Comment:
This allows me to use the `find_packages` function.  `setuptools` is now a requirement but in almost all cases, everyone has it so its mostly a no-op change.